src/diffusers/loaders/single_file_utils.py

```diff
@@ -178,7 +178,7 @@
 LDM_UNET_KEY = "model.diffusion_model."
 LDM_CONTROLNET_KEY = "control_model."
 LDM_CLIP_PREFIX_TO_REMOVE = ["cond_stage_model.transformer.", "conditioner.embedders.0.transformer."]
-LDM_OPEN_CLIP_TEXT_PROJECTION_DIM = 1024
+LDM_OPEN_CLIP_TEXT_PROJECTION_DIM = 1280
```

Comment:
Went looking everywhere to find why this is 1024 but couldn't. 

* https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0/blob/main/text_encoder_2/config.json has 1280 as `projection_dim`.
* https://huggingface.co/laion/CLIP-ViT-H-14-laion2B-s32B-b79K/blob/main/config.json#L115 as well.

Reply:
Wait. I might have found a better way of dealing with this. Pushing the changes in a few.

Reply:
This was the fallback value when we were using `download_from_original_stable_diffusion_ckpt`
https://github.com/huggingface/diffusers/blob/84905ca7287876b925b6bf8e9bb92fec21c78764/src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py#L960

Reply:
I believe it might be for SD 2
https://huggingface.co/stabilityai/stable-diffusion-2-1/blob/main/text_encoder/config.json